client/functions.lua

```diff
@@ -505,10 +486,9 @@ function QBCore.Functions.GetGroundZCoord(coords)
 
     local retval, groundZ = GetGroundZFor_3dCoord(coords.x, coords.y, coords.z, false)
     if retval then
-        return vector3(coords.x, coords.y, groundZ)
+        return vec3(coords.x, coords.y, groundZ)
     else
-        print('Couldn\'t find Ground Z Coordinates given 3D Coordinates')
-        print(coords)
+        print('Couldn\'t find Ground Z Coordinates given 3D Coordinates:', coords)
         return coords
     end
```

Comment:
You can remove this else statement and drop the block to the lower indentation level. It functions the same.